internal/collector/telemetry/telemetry.go

```diff
@@ -57,10 +57,10 @@ func Flags(flags *flag.FlagSet) {
 
 	// At least until we can use a generic, i.e.: OpenCensus, metrics exporter
 	// we default to Prometheus at port 8888, if not otherwise specified.
-	metricsPortPtr = flags.Uint(
-		metricsPortCfg,
-		8888,
-		"Port exposing collector telemetry.")
+	metricsAddrPtr = flags.String(
+		metricsAddrCfg,
+		"localhost:8888",
```

Comment:
> If changing the default isn't desired it can stay as before. 

Let's use your concept of "dev" build for the other PR to decide on the default: current default for non-dev releases.